docs/user-guide/agile-boards/README.md

```diff
@@ -39,22 +41,44 @@ The new Boards are tightly integrated with all other project management function
 
 ## Create a new Board
 
-You can create as many flexible boards in a project as you need and configure them to your needs. First, you need to create a new Boards view. 
-If you haven't done so yet, [activate the Boards module](../projects/project-settings/modules) within your project. Also, we recommend to verify [Roles and Permissions](https://www.openproject.org/help/administration/manage-roles-permissions/) within your system's Administration.
+You can create as many Agile Boards in a project as you need and configure them to your needs. First, you need to create a new Boards view. 
```

Comment:
We should ideally use one way of capitalizing agile boards (either Agile boards (as in the beginning of the page) or Agile Boards (capitalized)).